packages/core/src/Tab/Tab.js

```diff
@@ -82,6 +82,11 @@ const Tab = ({
                 overflow: hidden;
                 text-overflow: ellipsis;
                 transition: fill 150ms ease-in-out;
+                padding: 0px 2px;
```

Comment:
What's the purpose of this padding? Removing it doesn't seem to affect the focus styles, and I'd rather not make aesthetic changes unless we need to.

Reply:
I thought the extra padding was being suggested by the description of https://jira.dhis2.org/browse/LIBS-77 (maybe to make the outline less tight around the text?). I'll remove it :+1:

Reply:
Oh yeah, sorry! It was actually me who initially suggested it 😅 I think let's keep it out to avoid the style change 👍